app/views/users/profile.py

```diff
@@ -63,6 +63,11 @@ def connect_instagram():
     instagram_auth_url, state = instagram.authorization_url(InstagramOAuth.get_auth_uri(), access_type='offline')
     return redirect(instagram_auth_url)
 
+@profile.route('/google_connect', methods=('GET', 'POST'))
```

Comment:
@gaeun we already have a callback for google at https://github.com/fossasia/open-event-orga-server/blob/development/app/views/utils_routes.py#L24 . Please reuse the same callback.